modules/sovrnBidAdapter.js

```diff
@@ -30,6 +30,7 @@ export const spec = {
       let eids;
       let tpid = []
       let criteoId;
+      let imp;
```

Comment:
Imp is not used outside of the _each scope so it should be defined there.

Reply:
Done!